src/tlo/core.py

```diff
@@ -445,19 +445,19 @@ def do_at_generic_first_appt(
         :param treatment_id: The treatment id of the HSI event triggering the generic appointment.
         :param random_state: Random number generator to be used when making random choices during event creation.
         """
-        return {}
+        return None
```

Comment:
```suggestion
        return
```

As we're recommending to not use `return None` in the docstring. I think you could also just omit the `return` statement entirely here (as the docstring is sufficient for this to qualify as valid function definition with indendented body) which would align with the prefered style in docstring.